bullet_train/app/controllers/concerns/account/onboarding/invitation_lists/controller_base.rb

```diff
@@ -0,0 +1,67 @@
+module Account::Onboarding::InvitationLists::ControllerBase
+  extend ActiveSupport::Concern
+
+  included do
+    layout "devise"
+
+    before_action do
+      @user = current_user
+    end
+  end
+
+  def new
+    @account_onboarding_invitation_list = Account::Onboarding::InvitationList.new
+  end
+
+  def create
+    @account_onboarding_invitation_list = Account::Onboarding::InvitationList.create(account_onboarding_invitation_list_params)
+
+    # Set default values for invitations and memberships.
+    # `save` below checks if the values are valid or not.
+    @account_onboarding_invitation_list.team = current_team
+    @account_onboarding_invitation_list.invitations.each_with_index do |invitation, idx|
+      invitation.team = current_team
+      invitation.from_membership = current_membership
+      invitation.membership.team = current_team
+
+      # Role IDs don't get registered automatically because roles_ids is an array, so we handle that here.
+      if available_roles.include?(params[:account_onboarding_invitation_list][:invitations_attributes][idx.to_s][:membership_attributes][:role_ids].downcase)
```

Comment:
We could probably make the params call here shorter, will try to keep this one in the back of my mind.